### PR TITLE
Consul sintax changes

### PR DESCRIPTION
**Pull request by gituser173**
Consul, marathon and powershell syntax changes

Signed-off-by: Claudiu Ban claudiu-laurian.ban@hpe.com

### CODE REVIEW

File: content/io/cloudslang/consul/create_kv.sl [hunk 1]
```diff
@@ -31,23 +31,23 @@ operation:
...
-        default: "'0'"
+        default: "0"
         required: false
     - body:
-        default: "key_value"
+        default: ${key_value}
         overridable: false
     - url:
-        default: "'http://'+ host + ':' + consul_port +'/v1/kv/'+key_name+'?flags='+flags"
+        default: "${'http://'+ host + ':' + consul_port +'/v1/kv/'+key_name+'?flags='+flags}"
```
Comment (Bonczidai) [inline content/io/cloudslang/consul/create_kv.sl]: I don't think you need surrounding quotes here - please verify (all the places on this changelist - do a search after `"$`)

**Comment by Bonczidai** [comment]
:+1: 

please look into the quotes issue

File: content/io/cloudslang/consul/create_kv.sl (hunk 1 above)
Comment (gituser173) [inline content/io/cloudslang/consul/create_kv.sl]: @mihaitusa told me there is a problem where there is ':' inside if I remove the surrounding quotes, I will check

Comment (Bonczidai) [inline content/io/cloudslang/consul/create_kv.sl]: only in some cases, you should check every time and use quotes only if necessary

**Comment by Bonczidai** [comment]
looks pretty good!

**Pull request merged**